stack-qa/recipes/deploy-vagrant-ci.rb

```diff
@@ -55,6 +55,26 @@
   end
 end
 
+# START elements to setup git_updater
+# Install git_update_scrupt.sh from files directory
+cookbook_file '/opt/vagrant/bin/update_github_sources.sh' do
+  source 'update_github_sources.sh'
+  owner deploy_user_name
+  group deploy_user_name
+  mode '0755'
+  action :create
+end
+
+# setup git_update_script cron to run as root, script will pick up user with git creds
+cron 'cookbooks_report' do
```

Comment:
@gilleyj Can you use the `cron_d` provider thingy? It'll create this in `/etc/cron.d/` as a job vs. editing the crontab directly.

https://github.com/till/easybib-cookbooks/tree/master/cron#resources-and-providers